docker-compose-dev.yml

```diff
@@ -80,3 +80,11 @@ services:
       - ./dev/faasm-local:/usr/local/faasm
       - /var/run/docker.sock:/var/run/docker.sock
       - /usr/bin/docker:/usr/bin/docker
+
+  vector:
+    image: netflixoss/vector:latest
+    depends_on:
+      - faasm-cli
+    network_mode: host
+    ports:
+      - "80:80"
```

Comment:
This won't run unless it's needed. I found it super useful to have around and I think it should be our first point of call when doing any profiling just to get a high-level overview of what's going on.